src/main/java/de/tum/in/www1/artemis/service/FileService.java

```diff
@@ -1018,14 +1018,18 @@ public String getUniquePathString(String path) {
         return getUniquePath(path).toString();
     }
 
+    public String getTemporaryUniquePathString(String path, long deleteDelayInMinutes) {
```

Comment:
Don’t think this is necessary? In the cases where it is used, it is used like
```java
Path.of(getTemporaryUniquePathString(...), someOtherString)
```
This could be rewritten to
```java
getTemporaryUniquePath(...).resolve(someOtherString)
```